README.md

```diff
@@ -6,5 +6,12 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
-1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
-1. Describe the flow of a PATCH request as it goes through the app.
+Rails is a web framework written in Ruby that allows people to build web application. It is used for the backend of the server to handle client request, application logic, database communication, and so on.
+
+2. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+see submissions folder
```

Comment:
I was looking for a diagram of how the different pieces interconnect, but your explanation makes it clear that you understand.